### PR TITLE
More material

### PR DESCRIPTION
**Pull request by hzsweers**
Material dialog migration is finished

List dividers are gone for the most part

Ripples and standard selectors

### CODE REVIEW

**Comment by hzsweers** [comment]
@Meisolsson @fadils want to take a look?

File: app/src/main/res/layout/commit_diff_line.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     android:id="@+id/tv_diff"
     style="@style/CodeText"
     android:layout_width="match_parent"
-    android:background="@drawable/list_item_background"
```
Comment (larsgrefer) [inline app/src/main/res/layout/commit_diff_line.xml]: @hzsweers You've replaced many references to this drawable. Should the drawable itself be removed too, or are there still references to the drawable left?

Comment (hzsweers) [inline app/src/main/res/layout/commit_diff_line.xml]: Still used in some places

**Comment by Meisolsson** [comment]
LGTM

**Pull request merged**